### DISCUSSION TITLE
Icelandic Language Leader

### DISCUSSION THREAD

**Discussion by thdg** [OHF-Voice/intents#1284] [Language Leaders]
Hi,
there is another discussion (#420) about an Icelandic LL that did not seem to go anywhere.
I want to apply to be a language leader.
I work at the University of Reykjavík in Language Technology focusing on TTS and ASR.
There is an pull request (#1255) from me that has not been reviewed.

**Comment by synesthesiam** [comment]
Thanks @thdg, I will take a look!

**Reply by synesthesiam** [reply]
Adding you as a language leader now :slightly_smiling_face:
